scalpel/src/Text/HTML/Scalpel/Internal/Scrape/URL.hs

```diff
@@ -57,36 +57,31 @@ instance TagSoup.StringLike str => Default.Default (Config str) where
 -- The default behavior is to use the global manager provided by
 -- http-client-tls (via 'HTTP.getGlobalManager'). Any exceptions thrown by
 -- http-client are not caught and are bubbled up to the caller.
-scrapeURL :: (TagSoup.StringLike str)
-          => URL -> Scraper str a -> IO (Maybe a)
+-- scrapeURL :: URL -> Scraper Text a -> IO (Maybe a)
```

Comment:
Is there something that makes it hard to type annotate this? Or is this just temporarily commented out while you change broader types around?